Makefile

```diff
@@ -340,13 +336,9 @@ endif
 # Protobufs #
 #############
 
-protos: $(PROTO_GOS)
-
-# use with care. This signals to make that the proto definitions don't need recompiling.
-touch-protos:
-	for proto in $(PROTO_GOS); do [ -f "./$${proto}" ] && touch "$${proto}" && echo "touched $${proto}"; done
+protos: clean-protos $(PROTO_GOS)
 
-%.pb.go: $(PROTO_DEFS)
+%.pb.go:
```

Comment:
depending on the proto definitions triggers Makes desire to compare timestamps of file before choosing to do anything, we always want to build protos every time.